lib/angle.rb

```diff
@@ -1,305 +1,332 @@
-require_relative 'core_extensions.rb'
-
-#Class Angle is a utility class that allows
-# * Angle arithmetic ( +,-,*,/,**,% and unary +,-)
-# * Angle comparison ( <=>, hence, <, >, >=, <=, == )
-# * Conversion to and from degrees and radians
-# * Conversion to string as radians or DMS format
-
-class Angle
-  include Comparable
-
-  # @return [Float] stored in radians
-  attr_accessor :angle
-  alias :value :angle    #Older version of angle used volue rather than angle
-  alias :value= :angle=  #Older version of angle used volue rather than angle
-
-  # @param [#to_f,#to_radians] angle may be anything that has a to_f and to_radians.
-  # @param [true,false, :radians] radians Angle is in degrees unless radians == true (or set to :radians).
-  def initialize(angle=0, radians=false)
-    #assumes that we are getting an angle in degrees.
-    if radians == true || radians == :radians
-      @angle = angle.to_f #works for String, Fixed, other Angles and for Float.
-    else
-      if angle.class == Array
-        @angle = self.class.decimal_deg(*angle).to_radians #Wild assumption that the array is deg,min,sec.
+require_relative 'core_extensions'
+
+module Vincenty
+
+  #Class Angle is a utility class that allows
+  # * Angle arithmetic ( +,-,*,/,**,% and unary +,-)
+  # * Angle comparison ( <=>, hence, <, >, >=, <=, == )
+  # * Conversion to and from degrees and radians
+  # * Conversion to string as radians or DMS format
+
+  class Angle
+    include Comparable
+
+    # @return [Float] stored in radians
+    attr_accessor :angle
+    alias :value :angle    #Older version of angle used volue rather than angle
+    alias :value= :angle=  #Older version of angle used volue rather than angle
+
+    # @param [#to_f,#to_radians] angle may be anything that has a to_f and to_radians.
+    # @param [true,false, :radians] radians Angle is in degrees unless radians == true (or set to :radians).
+    def initialize(angle=nil, radians=false)
+      #assumes that we are getting an angle in degrees.
+      if radians == true || radians == :radians
+        if angle.is_a? Angle
+          temp_angle = angle.angle
+        else
+          temp_angle = (angle || 0).to_f #works for String, Fixed, and Float.
+        end
       else
```

Comment:
The addition of indentation (via wrapping this in a module) makes the diff look like there are logic changes because it finds a lot of `else`s and `end`s that seem to line up.  That's not the case here; only `to_d` and `to_r` references would have been changed.